src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -406,6 +409,9 @@ public static void Verify_InvalidTargetUnsafeAccessor()
         [UnsafeAccessor(UnsafeAccessorKind.Field, Name=DoesNotExist)]
         extern static ref string FieldNotFound(UserDataClass d);
 
+        [UnsafeAccessor(UnsafeAccessorKind.Field, Name=UserDataClass.StaticFieldName)]
+        extern static ref string FieldNotFoundStaticStatus(UserDataClass d);
```

Comment:
nit: maybe `FieldNotFoundStaticMismatch` - "status" is kind of unclear

Reply:
nit2: The problematic case for Mono was the case where the target type is a `struct`, not `class`

Reply:
Right, will change it to valuetype

Reply:
Changed.